src/Billing/Controllers/StripeController.cs

```diff
@@ -501,27 +507,29 @@ private async Task<bool> ValidateCloudRegionAsync(Event parsedEvent)
             case HandledStripeWebhook.SubscriptionUpdated:
                 {
                     var subscription = await GetSubscriptionAsync(parsedEvent, true, new List<string> { "customer" });
-                    customerRegion = GetCustomerRegionFromMetadata(subscription.Customer.Metadata);
+                    customerRegion = GetCustomerRegionFromMetadata(subscription.Customer?.Metadata);
```

Comment:
The look Great. Is it possible that subscription object will be null? Can we check for null